languages/wikipron/src/whitelist.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env python
+
+import sys
```

Comment:
Sort imports lexicographically; e.g., import `re` before `sys`.

FYI, the best-practice rules for imports are:

* imports from the standard library (except `typing`), in lexicographic order, using the `import X` style
* a blank line
* imports from `typing`, using the `from typing import X` style
* a blank line
* imports from third-party libraries (e.g., installable from pip), in lexicographic order, using the `import X` style
* a blank line
* any module-specific imports, in lexicographic order, using the `import X` style

(You can omit the blank lines between blocks if any of those blocks are empty.)